src/Pmi/Review/Review.php

```diff
@@ -0,0 +1,178 @@
+<?php
+namespace Pmi\Review;
+
+use Pmi\Order\Order;
+use Pmi\Evaluation\Evaluation;
+
+class Review
```

Comment:
Since most of these methods require the database, would it make sense to pass in the db through the constructor?